compiler-plugin/src/main/kotlin/arrow/meta/MetaPlugin.kt

```diff
@@ -18,7 +18,7 @@ open class MetaPlugin : Meta {
     listOf(
       //higherKindedTypes2,
       //typeClasses,
-      comprehensions,
+      // comprehensions,
```

Comment:
@nomisRev , build check is failing because comprehensions test should be disabled because of this change.

Reply:
Thanks @i-walker :raised_hands: